### PR TITLE
jenkins/pipelines: add whitelist for mysql_test in master branch

### PR DESCRIPTION
**Pull request by bb7133**

### CODE REVIEW

File: jenkins/pipelines/ci/tidb/tidb_ghpr_integration_common_test.groovy [hunk 1]
```diff
@@ -565,7 +565,11 @@ try {
 
             tests["Integration MySQL Test"] = {
                 try {
-                    run("mysql_test", "mysqltest", "./test.sh")
+                    if TIDB_TEST_BRANCH == "master" {
```
Comment (purelind) [inline jenkins/pipelines/ci/tidb/tidb_ghpr_integration_common_test.groovy:568]: Some grammar mistakes here. ` if TIDB_TEST_BRANCH == "master" {` 
correct way:  `if (TIDB_TEST_BRANCH == "master") {`

**Comment by purelind** [comment]
replay ci to test tidb_ghpr_mysql_test is ok
https://ci.pingcap.net/blue/organizations/jenkins/tidb_ghpr_mysql_test/detail/tidb_ghpr_mysql_test/9384/pipeline/

File: jenkins/pipelines/ci/tidb/tidb_ghpr_integration_common_test.groovy (hunk 1 above)
Comment (bb7133) [reply jenkins/pipelines/ci/tidb/tidb_ghpr_integration_common_test.groovy]: Addressed, PTAL

**Comment by purelind** [comment]
replay test
1. https://ci.pingcap.net/blue/organizations/jenkins/tidb_test_ghpr_mysql_test/detail/tidb_test_ghpr_mysql_test/668/pipeline/76/
2. https://ci.pingcap.net/blue/organizations/jenkins/tidb_ghpr_integration_common_test/detail/tidb_ghpr_integration_common_test/11345/pipeline/
3. https://ci.pingcap.net/blue/organizations/jenkins/tidb_ghpr_mysql_test/detail/tidb_ghpr_mysql_test/9525/pipeline/

**Review by purelind — APPROVED** [review]

**Pull request merged**